t3f/ops.py

```diff
@@ -1132,3 +1136,50 @@ def gather_nd(tt, indices):
     tt_elements = tf.matmul(tt_elements, core_slices)
   tt_elements = tf.reshape(tt_elements, tf.shape(indices)[:-1])
   return tt_elements
+
+def renormalize_cores(tt):
```

Comment:
maybe better renormalize_tt_cores? I tried not to say the word 'cores' (and use tt-cores instead) anywhere but in the internal code.